src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala

```diff
@@ -29,7 +29,7 @@ class ApiValidationSpec
     with BeforeAndAfterEach {
 
   val kubernetesClient: KubernetesClient = k8sInit
-  val stackSetOperations     = new StackSetOperations(kubernetesClient)
+  val stackSetOperations = new StackSetOperations(kubernetesClient)
```

Comment:
Is this being formatted using the scalafmt config? I think that usually tries to line up definitions